.github/pull_request_template.md

```diff
@@ -11,12 +11,15 @@ Check options that are relevant.
 - [ ] Fix or feature that would cause existing functionality to not work as expected
 - [ ] This change requires a update in the design document
 
-**Checklist:**
+**Steps to Verify Functionality**
```

Comment:
Felt this was needed since there were some occasions when steps to verify things were fixed or working were requested.